packages/core/src/filesystem/authConfig.ts

```diff
@@ -25,7 +32,9 @@ export function getOrCreateAuthConfig(directory?: string): AuthConfig {
   const configPath = PrismicConfigPath(directory);
   if (!Files.exists(configPath)) return createDefaultAuthConfig(directory);
 
-  const conf = Files.readJson(configPath);
+  const conf = Files.safeReadEntity(configPath, (payload) => {
+    return getOrElseW(() => null)(AuthConfig.decode(payload));
```

Comment:
The getOrElse must have a default value that has the same type as the one if right, getOrElseW allows you to return a value of a different type, null here.

Reply:
We used to not have the config validated, that's why we had to spread, make sure we had the good type in the end.

Here you validate, so you could return the default if it doesn't valide and there would be no extra operation of spreading in case of a right file